static/templates/authentication/forgot-password.html

```diff
@@ -0,0 +1,17 @@
+<div  layout="row" flex="50" layout-align="center center">
+    <md-content class="authentication-form" layout-padding class="md-gray-theme" flex="50" layout-align="center center" layout="column">
+        <h3 id="form-header">Forgot Password</h3>
+        <form name="form">
+            <div layout="row">
+                <md-input-container>
+                    <label>Email</label>
+                    <input type="email" required ng-model="auth.email">
+                </md-input-container>
```

Comment:
after I submit the email address, i am on the same page, we should show something like "email sent" in a toast message or on the form.